src/credentials_sample.h

```diff
@@ -12,6 +12,9 @@ class CREDENTIALS : public QObject
         return "UA-XXXXXXXX-X";
     }
 
+    static const QString GA_SENTRY_DSN() {
+        return "https://a123:b456@sentry.io/c789";
```

Comment:
How will we embed the Sentry credentials for the production build?

Reply:
We can grab the `credentials.h` file from 1password Dev: LotusValley vault and use it for build the release version